### PR TITLE
RHDEVDOCS-2697

### PR DESCRIPTION
**Pull request by rolfedh**
"do not configure the Log Forwarding API" statement is too general.
https://issues.redhat.com/browse/RHDEVDOCS-2697
[enterprise-4.6][enterprise-4.7][enterprise-4.8]

### CODE REVIEW

**Comment by jtudelag** [comment]
@rolfedh this is much more clear now, thanks!!

But I think there is room for clarification (avoid contrary statements), [here](https://github.com/openshift/openshift-docs/blob/a7a67662173daacc569af001a3c45c2ef310f20c/modules/cluster-logging-collector-log-forwarding-about.adoc#L10) it is mentioned as well: 

`If you want to forward all logs to only the internal OpenShift Container Platform Elasticsearch instance, do not configure the Log Forwarding API.`

**Comment by rolfedh** [comment]
> @rolfedh this is much more clear now, thanks!!
> 
> But I think there is room for clarification (avoid contrary statements), [here](https://github.com/openshift/openshift-docs/blob/a7a67662173daacc569af001a3c45c2ef310f20c/modules/cluster-logging-collector-log-forwarding-about.adoc#L10) it is mentioned as well:
> 
> `If you want to forward all logs to only the internal OpenShift Container Platform Elasticsearch instance, do not configure the Log Forwarding API.`

Thanks for spotting that one, too. Updated.

File: logging/cluster-logging-external.adoc [hunk 1]
```diff
@@ -5,19 +5,18 @@ include::modules/common-attributes.adoc[]
...
 
-Alternatively, you can create a ConfigMap to use the xref:../logging/cluster-logging-external.html#cluster-logging-collector-legacy-fluentd_cluster-logging-external[Fluentd *forward* protocol] or the xref:../logging/cluster-logging-external.html#cluster-logging-collector-legacy-syslog_cluster-logging-external[syslog protocol] to send logs to external systems. However, these methods for forwarding logs are deprecated in {product-title} and will be removed in a future release. 
+When you forward logs externally, the Cluster Logging Operator creates or modifies a Fluentd config map to send logs using your desired protocols. You are responsible for configuring the protocol on the external log aggregator.
+
+Alternatively, you can create a ConfigMap to use the xref:../logging/cluster-logging-external.html#cluster-logging-collector-legacy-fluentd_cluster-logging-external[Fluentd *forward* protocol] or the xref:../logging/cluster-logging-external.html#cluster-logging-collector-legacy-syslog_cluster-logging-external[syslog protocol] to send logs to external systems. However, these methods for forwarding logs are deprecated in {product-title} and will be removed in a future release.
```
Comment (ahardin-rh) [inline logging/cluster-logging-external.adoc:19]: s/ConfigMap/config map

File: modules/cluster-logging-collector-log-forwarding-about.adoc [hunk 2]
```diff
@@ -30,7 +30,7 @@ If the output URL scheme requires TLS (HTTPS, TLS, or UDPS), then TLS server-sid
 
 * `infrastructure`. Container logs from pods that run in the `openshift*`, `kube*`, or `default` projects and journal logs sourced from node file system.
 
-* `audit`. Logs generated by the node audit system (auditd) and the audit logs from the Kubernetes API server and the OpenShift API server. 
+* `audit`. Logs generated by the node audit system (auditd) and the audit logs from the Kubernetes API server and the OpenShift API server.
```
Comment (ahardin-rh) [inline modules/cluster-logging-collector-log-forwarding-about.adoc:33]: the node audit system, called auditid, and the 

(We try to avoid parentheticals.)

**Comment by ahardin-rh** [comment]
Just a few minor style suggestions from me! :+1:

**Comment by rolfedh** [comment]
@anpingli Would you review and mark this "lgtm?" Thank you.

**Review by anpingli — COMMENTED** [review]
/lgtm

**Pull request merged**